### PR TITLE
xfailing test for #568

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
.

### CODE REVIEW

File: testing/test_mark.py [hunk 1]
```diff
@@ -418,6 +418,31 @@ def test_bar(self): pass
...
+            @pytest.mark.b
+            class TestSub(TestBase):
+                pass
+
+
+            class TestOtherSub(TestBase):
+                pass
+
+        """)
+        (base_item, sub_item, sub_item_other), rec = testdir.inline_genitems(p)
```
Comment (The-Compiler) [inline testing/test_mark.py]: Is this (nested unpacking) supported on 2.6?

**Comment by The-Compiler** [comment]
#586 is "python.raises examples are unclear" - please amend the commit to point to the right issue.

File: testing/test_mark.py [hunk 2]
```diff
@@ -418,6 +418,32 @@ def test_bar(self): pass
         items, rec = testdir.inline_genitems(p)
         self.assert_markers(items, test_foo=('a', 'b'), test_bar=('a',))
 
+
+    @pytest.mark.issue568
+    @pytest.mark.xfail(reason="markers smear, needs fixing")
+    def test_mark_decorator_subclass_does_not_propagate_to_base2(self, testdir):
```
Comment (omarkohl) [inline testing/test_mark.py]: Name of the test is probably wrong

**Comment by nicoddemus** [comment]
@RonnyPfannschmidt could you please fix the PR as people mentioned and push an amended version? Thanks! :grin:

**Pull request merged**